lib/parse/credit.js

```diff
@@ -13,13 +13,13 @@ export default function parseCredit({ dueDate, finishDate }) {
   return computeCredit(timeDiff({ dueDate, finishDate }))
 }
 
-export const calculateReliability = (promises) =>
-  _(promises)
-    .map((p) => p.credit)
-    .compact()
-    .mean()
+export function calculateReliability(promises) {
 
-// Return the count of promises included in the reliability calculation.
-// Right now, that is all promises completed or overdue - "due or done"
-export const promisesIncluded = (promises) => 
-  _(promises).map((p)=>p.credit).compact().size()
\ No newline at end of file
+  const pScores = _(promises)
+                 .map((p) => p.credit)
+                 .compact()
+  const score = pScores.mean()
+  const counted = pScores.size()
+
+  return {score: score, counted: counted}
```

Comment:
nice refactor!

with es6 the return could be shortened to:

```js
	const score = pScores.mean()
	const counted = pScores.size()
 	return { score, counted }
```

but it might be okay to just do this instead:

```js
	return {
		score: pScores.mean(),
		counted: pScores.size()
	}
```

Reply:
I'll go with the first version, so when I watch it in the interactive debugger I can see what score and counted are before we return them from the function.